cmd/eelbot/main.go

```diff
@@ -43,30 +46,70 @@ func main() {
 		panic("no token provided; please use -t to provide bot token")
 	}
 
+	if configFile == "" {
```

Comment:
No need to trim this right?

Reply:
I don't believe so, since it should only be nonempty when its set via a flag. The only way to have leading or trailing whitespace is to explictly quote it, for ex: `./eelbot -c "   "`